src/components/MostPopular/ProductItem/ProductItem.js

```diff
@@ -125,7 +138,12 @@ class ProductItem extends Component {
       <Modal className={`${CN}-quick-view`} removeModal={this.removeModal}>
         <Heading title="Quick view" />
         <div className={`${CN}-quick-view-content`}>
-          <ProductImage className={`${CN}-quick-view-content-images`} images={imagesForQuickView} verticalCarousel />
+          <ProductImage
+            className={`${CN}-quick-view-content-images`}
```

Comment:
Please take a look at the BEM naming convention: http://getbem.com/naming/ . 
`className={`${CN}-quick-view-content-images`}` should be `className={`${CN}__quick-view__content__images`}`, because it goes like this: `block__element__element--modifier` (modifier is something like 'hidden', 'disabled' and similar, it's not an element name but it is its state characteristic).

Reply:
done)

Reply:
it's not fixed yet